src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -130,8 +138,11 @@ protected function initialize()
         }
     }
 
-    private function addResourceFiles($filesByLocale)
+    private function addResourceFiles()
     {
+        $filesByLocale = $this->resourceFiles;
```

Comment:
With this assignment, the resources as cloned again, and now we have 3 times the same content for a moment. We already have $this->options['resource_files'] to iterate in the next line